pkgs/tools/networking/shadowsocks-v2ray-plugin/default.nix

```diff
@@ -3,6 +3,7 @@
 buildGoModule rec {
   pname = "shadowsocks-v2ray-plugin";
   version = "1.3.1";
+  # nixpkgs-update: no auto update
```

Comment:
I think we can comments with links to
https://github.com/NixOS/nixpkgs/issues/219343
https://github.com/shadowsocks/v2ray-plugin/issues/292

Reply:
Oh, I've merged it too fast. Sorry...